izpack-core/src/main/resources/com/izforge/izpack/bin/langpacks/installer/deu.xml

```diff
@@ -217,15 +218,16 @@
     <!-- PDFLicencePanel strings -->
     <str id="PDFLicencePanel.info" txt="Bitte lesen Sie die folgenden Lizenzvereinbarungen durch:"/>
 
-    <!-- PrinterSelect Panel strings -->
-    <str id="PrinterSelectPanel.select_printer" txt="Wählen Sie den Drucker für die erste Einrichtung und Prüfung."/>
-
     <!-- ProcessPanel strings -->
     <str id="ProcessPanel.heading" txt="Prozesse"/>
 
     <!-- Registry strings -->
     <str id="functionFailed.RegOpenKeyEx" txt="Der Registry-Schlüssel {0}\\{1} konnte nicht geöffnet werden."/>
 
+    <!-- Select Printer Panel strings -->
+    <str id="SelectPrinterPanel.no_printer" txt="No printers available to use for initial setup and testing."/>
```

Comment:
German translation is: "Kein Drucker für die erste Einrichtung und Prüfung verfügbar."

Reply:
Changed